xml/System.Net.Http/HttpContent.xml

```diff
@@ -1047,7 +1048,8 @@ When the `disposing` parameter is `true`, this method releases all resources hel
           <format type="text/markdown"><![CDATA[
 
 ## Remarks
- This operation will not block. The returned <xref:System.Threading.Tasks.Task%601> object will complete after all of the stream that represents content has been read.
+ This operation will not block. The returned <xref:System.Threading.Tasks.Task%601> object will complete after all of the stream that represents content has been read unless <xref:System.Net.Http.HttpContent.CreateContentReadStream(System.Threading.CancellationToken)> has been implemented to do otherwise.
```

Comment:
```suggestion
 This operation will not block. The returned <xref:System.Threading.Tasks.Task%601> object will complete after all of the stream that represents content has been read (unless <xref:System.Net.Http.HttpContent.CreateContentReadStream(System.Threading.CancellationToken)> has been implemented to do otherwise).
```